src/librustdoc/test.rs

```diff
@@ -116,8 +117,9 @@ pub fn run(options: Options) -> i32 {
             });
         });
 
-        Ok(collector.tests)
-    }).expect("compiler aborted in rustdoc!");
+        let ret : Result<_, ErrorReported> = Ok(collector.tests);
```

Comment:
Why was this variable added?

Reply:
Some type hint is required for the closure return. If left on line 88, the line becomes too long.

Reply:
Ah, ok.